driver/fixtures/fixtures_test.go

```diff
@@ -34,10 +39,17 @@ var Suite = &fixtures.Suite{
 	},
 }
 
+func SetPythonPath() {
+	os.Setenv("PYTHONPATH", filepath.Join(projectRoot,
```

Comment:
We can make `./build/bin/native` a bash script, set an env variable there and call `python_driver` from it.

Reply:
Yes, it actually is already a bash script, I'll change it, I was just testing things to make this work, but I fear the environment variable solution won't work for parsing when the Python2 binary must be called, maybe with .pth files...